demo-1-measure.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
File from another branch

Reply:
Correct. I'll remove this file from the PR.